### PR TITLE
[fx] skip diffusers unitest if it is not installed

### PR DESCRIPTION
**Pull request by feifeibear**

### CODE REVIEW

File: tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py [hunk 1]
```diff
@@ -16,6 +21,7 @@
 TIME_STEP = 2
 
 
+@pytest.mark.skipif(not HAS_DIFFUSERS, reason="diffusers has not be installed")
```
Comment (super-dainiu) [inline tests/test_fx/test_tracer/test_hf_model/test_hf_diffuser.py:24]: reason="``diffusers`` has not been installed"

**Comment by super-dainiu** [comment]
Should we remove diffusers from test_requirements.txt also? The CI has difficulties installing this.

**Comment by FrankLeeeee** [comment]
> Should we remove diffusers from test_requirements.txt also? The CI has difficulties installing this.

The standard operation procedure is that, when a developer wants to the do the test, he/she should do `pip install -r test-requirenements.txt` first before executing `pytest` in stead of making this test condititonal.

**Comment by FrankLeeeee** [comment]
> Should we remove diffusers from test_requirements.txt also? The CI has difficulties installing this.

**Comment by FrankLeeeee** [comment]
Sorry accidentally clicked on the `close with comment` button.

**Comment by feifeibear** [comment]
Let's set dependencies such as torchrec, diffuser, and triton as optional in the CAI unit test. In my opinion, our unit tests do not need to be coupled with them.
1. These libraries' interface updates are not under our control. 
2. Unitests need these libraries are bonus optimizatin, and their absence does not actually affect the core functions.

**Comment by FrankLeeeee** [comment]
> Let's set dependencies such as torchrec, diffuser, and triton as optional in the CAI unit test. In my opinion, our unit tests do not need to be coupled with them.
> 
> 1. These libraries' interface updates are not under our control.
> 2. Unitests need these libraries are bonus optimizatin, and their absence does not actually affect the core functions.

OK, this is fine. I can create another decorator to handle this case. Otherwise, every unit test file depending on these libraries will have redundant code 
```
try:
     import xxx
     HAS_XXX = True
else:
     HAS_XXX = False
```

**Comment by feifeibear** [comment]
why use else instead of except?

**Pull request merged**